test-support/build.gradle

```diff
@@ -0,0 +1,67 @@
+import org.gradle.util.GradleVersion
+
+plugins {
+    id 'java-library'
```

Comment:
I'm not sure, but maybe we only need the `java` plugin here.

Reply:
Yeah, we can remove the `maven-publish` plugin, since we're not publishing.

Reply:
Fixed and force-pushed.